### PR TITLE
Widget | Add bottom padding to pages

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Add padding to bottom of pages in Widget flow

## Justify why they are needed

- Needed to be able to scroll past the bottom of the page

- I duplicated the code to avoid coming up with a layout abstraction for something so simple

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3569** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3569?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #3570** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3570?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #3571** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3571?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #3574** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3574?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #3577** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3577?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #3579** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3579?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
              * **PR #3580** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3580?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                * **PR #3581** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3581?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3579?utm_source=stack-comment).

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3569** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3569?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #3570** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3570?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #3571** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3571?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #3574** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3574?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #3577** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3577?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #3579** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3579?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
              * **PR #3580** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3580?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                * **PR #3581** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3581?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3579?utm_source=stack-comment).

File: apps/store/src/features/widget/SignPage.tsx [hunk 1]
```diff
@@ -225,6 +225,11 @@ export const SignPage = (props: Props) => {
   )
 }
 
+const Wrapper = styled(Space)({
```
Comment (alebedev) [inline apps/store/src/features/widget/SignPage.tsx:228]: Do you think it's repetitive enough to extract common part?

Comment (robinandeer) [reply apps/store/src/features/widget/SignPage.tsx:228]: While we build the flow, I don't think it's worth it -- but I think one of the first post-launch activities should be to refactor the code with a mindset of what we can reuse :)

**Comment by robinandeer** [comment]
### Merge activity

* **Nov 16, 2:28 PM**: [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3579) rebased this pull request as part of a merge.

**Pull request merged**